data/mods/Aftershock/items/magazine/25mm.json

```diff
@@ -0,0 +1,19 @@
+[
+  {
+    "id": "afs_25mm_mag",
+    "looks_like": "stanag30",
+    "type": "MAGAZINE",
+    "name": { "str": "25x40mm 40-round magazine" },
+    "description": "A 40-round magazine for use with 25x40mm caseless firearms.",
+    "weight": "340 g",
+    "volume": "2 L",
+    "price": 70000,
+    "price_postapoc": 1000,
+    "material": [ "superalloy", "plastic" ],
```

Comment:
This is the only gun or magazine made using superalloy?  I feel like we should either use more superalloy or not use it here.

Reply:
Ahhh, yeah more of the magazines should use it!